openhtf/util/multicast.py

```diff
@@ -101,12 +109,12 @@ def send(message,
          port=DEFAULT_PORT,
          ttl=DEFAULT_TTL,
          timeout_s=2):
-  """Sends a message on the given multicast socket and returns responses.
+  """Sends a message to the given multicast socket and returns responses.
```

Comment:
this might be better called `query`, since you don't just send, you also wait for responses?